Tests/LibCrypto/TestASN1.cpp

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2021, Ben Wiederhake <BenWiederhake.GitHub@gmx.de>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/StringView.h>
+#include <LibCrypto/ASN1/ASN1.h>
+#include <LibTest/TestCase.h>
+
+class DebuggableDateTime {
+public:
+    DebuggableDateTime(Core::DateTime datetime)
+        : m_datetime(datetime)
+    {
+    }
+    DebuggableDateTime(int year, int month, int day, int hour, int minute, int second)
+    {
+        m_datetime = Core::DateTime::create(year, month, day, hour, minute, second);
+    }
+
+    [[nodiscard]] bool operator==(DebuggableDateTime const& other) const
+    {
+        return m_datetime.year() == other.m_datetime.year()
+            && m_datetime.month() == other.m_datetime.month()
+            && m_datetime.day() == other.m_datetime.day()
+            && m_datetime.hour() == other.m_datetime.hour()
+            && m_datetime.minute() == other.m_datetime.minute()
+            && m_datetime.second() == other.m_datetime.second();
+    }
+    Core::DateTime const& datetime() const { return m_datetime; }
+
+private:
+    Core::DateTime m_datetime;
+};
```

Comment:
What does `DebuggableDateTime` add here? If it's the `operator==`, wouldn't that be a useful addition to `Core::DateTime` instead?

Reply:
Also the `AK::Formatter` for it, but I'll just push both into Datetime.h.